packages/config/src/forkConfig/index.ts

```diff
@@ -148,19 +148,23 @@ export function createForkConfig(config: ChainConfig): ForkConfig {
           return config.MAX_BLOBS_PER_BLOCK;
       }
 
+      const blobSchedule = this.getBlobParameters(epoch);
+
+      return blobSchedule !== null ? blobSchedule.MAX_BLOBS_PER_BLOCK : config.MAX_BLOBS_PER_BLOCK_ELECTRA;
+    },
+    getMaxRequestBlobSidecars(fork: ForkName): number {
+      return isForkPostElectra(fork) ? config.MAX_REQUEST_BLOB_SIDECARS_ELECTRA : config.MAX_REQUEST_BLOB_SIDECARS;
+    },
+    getBlobParameters(epoch: Epoch): BlobScheduleEntry | null {
       // Sort by epoch in descending order to find the latest applicable value
       const blobSchedule = [...config.BLOB_SCHEDULE].sort((a, b) => b.EPOCH - a.EPOCH);
```

Comment:
not belong to this change but we should not do a sort every time we call this function
instead of that ensure a sort order at creation time